### ISSUE TITLE
Provide i18n guidance on localization of DOMException's .message

### ISSUE THREAD

**Issue by marcoscaceres** [heycam/webidl#996]
An issue about how browser's should localize `DOMException`'s `.message` attribute came up as part of the Geolocation API:
https://github.com/w3c/geolocation-api/issues/50

In Geolocation, we have a `GeolocationPositionError`, which is similar to a `DOMException`. 

Anyway, it would be good to provide some general guidance, as `DOMException` is the largest source of developer-facing error messages in the platform (and then we can point to that for `GeolocationPositionError`).

**Comment by annevk** [comment]
I don't think we should localize these messages, except perhaps when exposed in the console. Eventually we should standardize on these messages as they can be a source of fingerprinting (depending on what data is exposed in them).

**Comment by marcoscaceres** [comment]
Agree, and good point about fingerprinting!
